### PR TITLE
Create data-analysis.md

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

**Comment by mmiladi** [comment]
I was not sure, how to include the uploaded .png screenshot files now are as plain text /images/protocols/*png

File: _protocols/data-analysis.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+BeerDecoded-Nanopore data analysis protocol
+
+
+# Software installation
+
+
+## MinKnow
+1. Follow the instructions here to install MinKnow. MinKnow is the interface software to the Minion handheld device, for configurations, sequencing initiation and transfer of the sequencing signals from the device to the computer.
```
Comment (bgruening) [inline _protocols/data-analysis.md]: We do have build a container to make this simpler isn't it ... maybe we need to link it here?

File: _protocols/data-analysis.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## Albacore basecaller (optional)
+Please follow the instruction at the Nanopore website.
+https://community.nanoporetech.com/protocols/albacore-offline-basecalli/
+
+
+
+
+## Kraken2
+Use the [Kraken2 galaxy wrapper](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkraken2%2Fkraken2%2F2.0.7_beta%2Bgalaxy0&version=2.0.7_beta%2Bgalaxy0&__identifer=nie0zwyf4f9)
```
Comment (bgruening) [inline _protocols/data-analysis.md]: Should that be the street science subdomain?

**Comment by bebatut** [comment]
@mmiladi I did some formatting there. Let me know if the changes are ok for you

**Comment by mmiladi** [comment]
Thanks much @bgruening @bebatut !  Looks ready to be merged.

**Pull request merged**